_starter-kit/_patterns/03-components/button/_button.scss

```diff
@@ -2,16 +2,17 @@
 // Styles for Button.
 
 .button {
+  @extend %button-font;
```

Comment:
Why an additional line for extending the button font?  For simplicity/consistency, shouldn't the button mixin take care of this?

Reply:
The goal of the extends, in general, was to have the means to capture both the responsive changes(font-size and margins) and the contextual padding needs ( [element] + h2 { padding-top: x} in a reusable way. Mixins allow for the former but not the latter. In this instance, %button-font handles the type while the mixin manages the button appearance and interaction. Not wedded to the approach but was trying to stay consistent with managing type separately